src/main/resources/spring/migration-bean.xml

```diff
@@ -156,6 +159,10 @@
         <constructor-arg type="java.lang.String" value="index" />
     </bean>
 
+    <bean id="namespaceFile" class="java.io.File">
+        <constructor-arg type="java.lang.String" value="src/test/resources/namespaces.properties"/>
```

Comment:
I moved those resources into test, and then updated the console output when a user doesn't provide an app context xml file to reference github.  I can't pull it in and spit it out at the user since we're no long on the classpath :(